pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -170,6 +171,29 @@ func (pl *VolumeBinding) podHasPVCs(pod *v1.Pod) (bool, error) {
 	return hasPVC, nil
 }
 
+func (pl *VolumeBinding) isSchedulableAfterPersistentVolumeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	_, newPV, err := util.As[*v1.PersistentVolume](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
+	logger = klog.LoggerWithValues(
+		logger,
+		"Pod", klog.KObj(pod),
+		"PersistentVolume", klog.KObj(newPV),
+	)
+
+	for _, vol := range pod.Spec.Volumes {
+		if vol.Name == newPV.Name {
```

Comment:
How is `pod.Spec.Volumes[*].Name` related to PV's name?

Reply:
Sorry, that was my mistake, so I've fixed it.